src/App.jsx

```diff
@@ -14,70 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {selectedGood === '' ? (
```

Comment:
```suggestion
      {!selectedGood ? (
```
empty string converts in false in ternary first part
better to reverse checking.
selectedGood ? <render text wiht selected good and clear button : <render No Goods selected text>